apps/etterlatte-behandling/src/main/kotlin/behandling/aktivitetsplikt/vurdering/AktivitetspliktAktivitetsgradDao.kt

```diff
@@ -58,6 +58,25 @@ class AktivitetspliktAktivitetsgradDao(private val connectionAutoclosing: Connec
             }
         }
 
+    fun hentNyesteAktivitetsgrad(sakId: Long): AktivitetspliktAktivitetsgrad? =
```

Comment:
På sikt må vi nok ha noe mer avansert for å hente aktivitetsgrad, siden det kan være aktivitetsgrad på et tidspunkt der den siste ikke er gjeldende, men det er aktivitetsgrad for en tidligere periode.

Litt tilsvarende for unntak også. Men jeg tror dette fungerer bra i starten.

Reply:
Enig, gjelder begge to, litt som jeg kommenterte over. Ref. at det blir mulig fra førstegangsbehandling i https://github.com/navikt/pensjon-etterlatte-saksbehandling/pull/4918 er det kanskje greit å fikse opp i den logikken først som sist.